src/main/java/com/music/wynk/models/User.java

```diff
@@ -1,24 +1,28 @@
 package com.music.wynk.models;
 
-import org.springframework.data.annotation.Id;
-import org.springframework.data.mongodb.core.mapping.Document;
-
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.util.ArrayList;
 import java.util.List;
 
-@Document(collection = "Users")
+@Entity
 public class User {
 
     @Id
     private String userId;
     private String userName;
     private String userDesc;
-    private List<Song> songs;
```

Comment:
Because i was unable to setup one to many relationship so in order to implement popularity functionality i'm just storing name of the songs in this list now